common/app/implicits/FaciaContentFrontendHelpers.scala

```diff
@@ -22,6 +24,19 @@ object FaciaContentFrontendHelpers {
       imageOverride.orElse(defaultTrailPicture)
     }
 
+    def mainYouTubeMediaAtom: Option[MediaAtom] =
+      for {
+       main <- faciaContent.properties.maybeContent.map(_.fields.main)
+       atoms <-  faciaContent.properties.maybeContent.flatMap(_.atoms)
+       document <- Some(Jsoup.parse(main))
```

Comment:
As discussed IRL with @TBonnin IRL, I have added this as a Todo as after some significant effort I still haven't been able to add `blocks` to the content model due to an issue deriving from the 22 fields case class limit.